cli/utils/ipfs/get.from.ipfs.ts

```diff
@@ -21,17 +21,28 @@ export interface IpfsManifestData {
 }
 
 export default function provideGetFromIpfs(
-    ipfsHttpClient: AxiosInstance
+    ipfsHttpClient: AxiosInstance,
+    agentId: string
 ): GetFromIpfs {
     assertExists(ipfsHttpClient, 'ipfsHttpClient')
+    assertIsNonEmptyString(agentId, 'agentId')
 
     return async function getIpfsByHash(metadataHash: string) {
-        const { data } = await ipfsHttpClient.get(`/ipfs/${metadataHash}`);
+        try {
+            const { data } = await ipfsHttpClient({url: `/ipfs/${metadataHash}`, baseURL: "https://ipfs.forta.network"});
```

Comment:
lets use dependency injection to inject a `fortaIpfsHttpClient` (which is configured to point to ipfs.forta.network)

Reply:
Updated.